fedimint-cli/src/client.rs

```diff
@@ -57,8 +59,17 @@ pub enum ClientCmd {
     Reissue { oob_notes: OOBNotes },
     /// Prepare notes to send to a third party as a payment
     Spend {
+        /// The amount of e-cash to spend
         #[clap(value_parser = parse_fedimint_amount)]
         amount: Amount,
+        /// If the exact amount cannot be represented, return e-cash of a higher
+        /// value instead of failing
+        #[clap(long)]
+        allow_overpay: bool,
+        /// After how many seconds we will try to reclaim the e-cash if it
+        /// hasn't been redeemed by the recipient
+        #[clap(long, default_value_t = 3600)]
```

Comment:
**thought:**  Is this automatic behavior? If so, isn't this a bit low for a default for something that might be passed in a DM or even piece of paper?

Reply:
Good point, what would you propose? A week?

Reply:
Sounds reasonable.

BTW. Maybe we should also `eprintln!` the deadline to warn/educate the user?

Reply:
could that be a `warn` level log? These are still shown and should be printed to stderr anyway (if not we should fix that).

Reply:
The way I think about it is: logging is for reporting what the program is doing, while `(e)println` for the actual output for the user. Not feeling too strongly about it one way or another.

Reply:
We need a third stream, stdlog! :laughing:

Reply:
In the CLI what would trigger this refund?

Reply:
Yep, basically anything that runs the SM executor for long enough. What I like to do to trigger this manually is create a LN invoice I don't intend to pay and then await it. We might want a more general purpose "run executor" command.